src/main/java/ToDo.java

```diff
@@ -0,0 +1,10 @@
+public class ToDo extends Task{
+    public ToDo(String description){
+        super(description);
+    }
+
+    @Override
+    public String printTask(){
+        return "[T]" + super.printTask();
+    }
```

Comment:
Maybe the naming of this method could be more appropriately changed to getTaskIdentity since the actual method itself is not printing anything.